README.md

```diff
@@ -52,6 +52,12 @@ the encrypted peer-to-peer Tox protocol.
 **[Mailing list] |**
 **IRC:** [#qtox@libera.chat]
 
+Come join us in the dev chat! We're running a GroupBot with Tox ID
+648BF2EEE794E94444B848F8FC6AD3BA029C9BC2649BA761EF556DA17F549022A8D7596E7DBA.
```

Comment:
Should these be `tox:`-links?

Reply:
Ok. It doesn't do anything on github, so I didn't, but if someone has a different markdown renderer, I guess it might work for them.